src/desktop/components/main_layout/header/index.styl

```diff
@@ -38,7 +38,7 @@
     height header-logo-icon-size
     width header-logo-icon-size
 #main-layout-header-center
-  padding 0 30px
+  padding 0 20px 0 30px
```

Comment:
Additional 10px will come either from user element or notifications element to the right.